helm/charts/keto/values.yaml

```diff
@@ -164,3 +164,13 @@ tracing:
 tolerations: []
 
 affinity: {}
+
+# Configure the probes for when the deployment is considered ready and ongoing health check
+livenessProbe:
```

Comment:
Could you move those under deployment? :)

Reply:
Ok, i'll adapt it but a deployment does not exist yet for in the keto chart although i would like to add it to be consistent with the other charts.